faunadb-scala/src/main/scala/faunadb/values/Codec.scala

```diff
@@ -145,11 +198,12 @@ object Decoder {
       }
   }
 
-  implicit object ByteArrayDecoder extends Decoder[Array[Byte]] {
+  implicit def EitherDecoder[A, B](implicit a: Decoder[A], b: Decoder[B], tagA: ClassTag[A], tagB: ClassTag[B]): Decoder[Either[A, B]] = new Decoder[Either[A, B]] {
```

Comment:
It would be best to not require the ClassTag, as there are occasions where it may not be available.